addons/resource/static/src/views/fields/many2many_avatar_resource/many2many_avatar_resource_field.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="resource.AvatarResourceMany2XAutocomplete" t-inherit="web.AvatarMany2XAutocomplete">
+        <xpath expr="//span[hasclass('o_avatar_many2x_autocomplete')]/img" position="before">
+            <i t-if="option.resourceType === 'material'" class="o_material_resource fa fa-wrench rounded text-center me-1"/>
+        </xpath>
+        <xpath expr="//span[hasclass('o_avatar_many2x_autocomplete')]/img" position="attributes">
+            <attribute name="t-if"></attribute>
+            <attribute name="t-elif">option.value</attribute>
```

Comment:
Hm... At this point, I wonder if we shouldn't have an empty t-if in super, just like we ccould have empty methods to be overridden.

Reply:
> Hm... At this point, I wonder if we shouldn't have an empty t-if in super, just like we ccould have empty methods to be overridden.

I don't really understand your comment here. There is already a t-if on the img tag in the template `web.AvatarMany2XAutocomplete`. Could you clarify?

Reply:
If I get it right (please correct me if I'm wrong), we have a t-if in super, and here, we want to transform it into a t-elif, and add a t-if before that.
IMO, it is a little sketchy: if the condition changes in super, the change won't be taken into account here, as we have duplicated it.
So, my suggestion (not perfect either) is to have, in super, `t-if="false" do nothing, t-elif="option.value" ...`, so that here, we can just put our condition in super's t-if.

Is it more clear? Am I missing something?

Reply:
> If I get it right (please correct me if I'm wrong), we have a t-if in super, and here, we want to transform it into a t-elif, and add a t-if before that. IMO, it is a little sketchy: if the condition changes in super, the change won't be taken into account here, as we have duplicated it. So, my suggestion (not perfect either) is to have, in super, `t-if="false" do nothing, t-elif="option.value" ...`, so that here, we can just put our condition in super's t-if.
> 
> Is it more clear? Am I missing something?

Yes, I get your point! I would suggest this then:
```suggestion
            <i t-if="option.resourceType === 'material'" class="o_material_resource fa fa-wrench rounded text-center me-1"/>
        </xpath>
        <xpath expr="//span[hasclass('o_avatar_many2x_autocomplete')]/img" position="attributes">
            <attribute name="t-if" add="&amp;&amp; option.resourceType !== 'material'" separator=" "/>
```
Modifying super would involves a diff in web. I think it's better if we can avoid it. What do you think?